src/Aimeos/Shop/Controller/JqadmController.php

```diff
@@ -257,6 +257,17 @@ protected function createAdmin() : \Aimeos\Admin\JQAdm\Iface
 		$context = app( 'aimeos.context' )->get( false, 'backend' );
 		$context->setI18n( app( 'aimeos.i18n' )->get( array( $lang, 'en' ) ) );
 		$context->setLocale( app( 'aimeos.locale' )->getBackend( $context, $site ) );
+		
+		$siteManager = \Aimeos\MShop::create( $context, 'locale/site');
+
+		$filter = $siteManager->filter();
+		$filter->add([ 'locale.site.code' => $site ]);
+    
+		$siteItem = $siteManager->search( $filter->slice( 0 ) );
+		$siteConfig = $siteItem->getConfig()->toArray();
```

Comment:
`getConfig()` already returns an array, so calling `toArray()` should fail

Reply:
I did initially try to just use $siteItem->getConfig() but config()->apply($siteConfig) complained that $siteConfig was a Map rather than an array. I think this was because I wasn't popping the config array off of the list returned from the filter/search.

I have just changed this to remove ->toArray() and it works perfectly.